devices/tuya.js

```diff
@@ -678,6 +679,7 @@ module.exports = [
             {vendor: 'Earda', model: 'EDM-1ZAB-EU'},
             {vendor: 'Earda', model: 'EDM-1ZBA-EU'},
             {vendor: 'Mercator Ikuü', model: 'SSWD01'},
+            {vendor: 'Moes', model: 'TS0601'},
```

Comment:
Is there any model number written on the device itself?

Reply:
It's 'ZS-USD'